### ISSUE TITLE
Fix manual sidecar inject doc after 0.8 template changes 

### ISSUE THREAD

**Issue by tiswanso** [istio/istio.github.io#1210]
The section referencing the sidecar-injector-configmap appears to be no longer valid for 0.8 as make installgen no longer generates the configmap's manifest.  

```
kube-inject can also be run without access to a running Kubernetes cluster. Create local copies of the injection and mesh configmap.

kubectl create -f install/kubernetes/istio-sidecar-injector-configmap-release.yaml \
    --dry-run \
    -o=jsonpath='{.data.config}' > inject-config.yaml

kubectl -n istio-system get configmap istio -o=jsonpath='{.data.mesh}' > mesh-config.yaml
```

The configmap approach was very useful for testing with different versions of istio-proxy params and/or different istio-proxy images than what is in the installed istioctl version.

**Comment by tiswanso** [comment]
@ayj  @sdake  -- here's the issue per our slack discussion

**Comment by ayj** [comment]
fixed by #1244
